python_modules/libraries/dagster-celery/dagster_celery/tasks.py

```diff
@@ -215,6 +217,116 @@ def _execute_step_k8s_job(
     return _execute_step_k8s_job
 
 
+def create_docker_task(celery_app, **task_kwargs):
+    @celery_app.task(bind=True, name='execute_step_docker', **task_kwargs)
+    def _execute_step_docker(
+        _self,
+        instance_ref_dict,
+        step_keys,
+        run_config,
+        mode,
+        repo_name,
+        repo_location_name,
+        run_id,
+        docker_image,
+        docker_creds,
```

Comment:
a little concerned about passing the docker password in the celery job submission but not obvious what better way to do it